synfig-studio/src/player/main.cpp

```diff
@@ -145,7 +140,9 @@ int main(int argc, char **argv)
 	RendererMap::const_iterator ri = renderers.find(renderer_name);
 	if (ri == renderers.end() || !ri->second) {
 		error("unknown renderer: %s", renderer_name.c_str());
-		print_renderers(renderers);
+		for(const auto& iter : synfig::rendering::Renderer::get_renderers()) {
+			std::cout << (iter.first).c_str() << " - " << iter.second->get_name() << std::endl;
+		}
```

Comment:
@ice0  do you think it should be `std::cerr` or even `synfig::error`? Or even list the valid renderers here on error?

Reply:
@veermetri05 ping! :) please change it to `std::cerr`

Reply:
Done!